accounts-journey/presentation/ui/AccountListItemTableCell.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  AccountListItemTableCell.swift
+//  AccountsJourney
+//
+//  Created by George Nyakundi on 26/10/2023.
+//
+
+import UIKit
+import Resolver
+import BackbaseDesignSystem
+
+final class AccountListItemTableCell: UITableViewCell, Reusable {
```

Comment:
Shouldn't there be a configuration option for the tableview cell? Where you can configure the labels and the icon?